template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/README.md.tmpl

```diff
@@ -37,7 +37,7 @@ contained in the following files:
 │   │
 │   ├── databricks.yml          <- databricks.yml is the root bundle file for the ML project that can be loaded by databricks CLI bundles. It defines the bundle name, workspace URL and resource config component to be included.
 │   │
-{{ if and (eq .input_include_feature_store `no`) (eq .input_include_mlflow_recipes `no`) -}}
+{{- if and (eq .input_include_feature_store `no`) (eq .input_include_mlflow_recipes `no`) }}
```

Comment:
I think this file works correctly before the change.
It's still correct after the change.

Reply:
Gotcha makes sense, but yeah consistency